dbotconf/internal/cmd.go

```diff
@@ -43,9 +43,13 @@ var (
 	}
 )
 
+const ignoreFlag = "ignore"
+
 func BuildAndExecute() error {
 	rootCmd.AddCommand(generateCmd)
 	rootCmd.AddCommand(verifyCmd)
 
+	rootCmd.PersistentFlags().StringSlice(ignoreFlag, nil, "directory paths to ignore")
```

Comment:
nit: this is a bit ambiguous, it's not really a list of directories but a list of glob patterns as seen here https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/26c6685bcd4658a83564db5fdea67de2c160cb88/internal/repo/repo.go#L74

so we probably want something like:

```suggestion
	rootCmd.PersistentFlags().StringSlice(ignoreFlag, nil, "glob patterns to ignore")
```

or change the implementation to do equality comparison after making paths absolute.